src/styles/blocks/nav.scss

```
@@ -0,0 +1,33 @@
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+  }
+
+  &__item {
+    margin-right: 10px;

```

```suggestion
    margin-right: 20px;

```
у макеті відступи по 20px